### ISSUE TITLE
Client logs should encode string in utf8

### ISSUE THREAD

**Issue by ritzalam** [bigbluebutton/bigbluebutton#5625]
Client logs are written in nginx logs in non-utf8 format which can't be parsed as JSON.

Here is an example log.

```
[2018-05-11T14:02:53+02:00] "2018/5/11 12:2:53.116 :: w_lh9npjbjsnhm 743c0f1fd0dcbc6e048cb7424fe077f42af48afd-1526038584041 :: [INFO] {"tags":["apps","connected"],"sessionToken":"ruo0v8hcr4vlaylo","waitForApproval":false,"user":{"externalMeetingId":"2f42f2eeb4583b3fc34e79fa855fc00160a1b89c-28-1","meetingId":"743c0f1fd0dcbc6e048cb7424fe077f42af48afd-1526038584041","meetingName":"\xD0\x92\xD0\xB5\xD0\xB1\xD0\xB8\xD0\xBD\xD0\xB0\xD1\x80 \xD0\xBF\xD0\xBE \xD0\x9F\xD1\x80\xD0\xB0\xD0\xB2\xD0\xB8\xD0\xBB\xD0\xB0\xD0\xBC \xD0\xB2\xD0\xB5\xD0\xB4\xD0\xB5\xD0\xBD\xD0\xB8\xD1\x8F \xD0\x9A\xD0\xB0\xD1\x81\xD1\x81\xD0\xBE\xD0\xB2...","username":"\xD0\x91\xD0\xBE\xD0\xBD\xD0\xB4\xD0\xB0\xD1\x80\xD0\xB5\xD0\xBD\xD0\xBA\xD0\xBE \xD0\x9D\xD0\xB0\xD1\x82\xD0\xB0\xD0\xBB\xD1\x8F","userId":"w_lh9npjbjsnhm"},"utcTime":1526040173116,"connections":{"screenshare":"","apps":"EEKRDFVX41EU0NM3","video":"","voice":""},"tzOffsetMin":-180,"tokenValid":true,"logCode":"validate_token_response_received"}"
```

**Comment by kepstin** [comment]
I believe this is some encoding added by nginx when saving to the log. When you parse the nginx log, you should reverse the escaping that nginx added before attempting to parse the json.
So, not really a bug, I think.

**Comment by ritzalam** [comment]
Says we can use escape=json but that available in nginx 1.11. We use 1.10.3

https://stackoverflow.com/questions/25049667/how-to-generate-a-json-log-from-nginx

**Comment by ritzalam** [comment]
Closing as the decoding can be done on the log processors.
